docs/faq.rst

```diff
@@ -228,17 +222,14 @@ Note that online data back-ups occur if you "register" your KA Lite installation
 If you only want to backup the database, locate the ``.kalite/database/`` folder
 and copy and restore that one.
 
-I skipped the step for downloading the assessment items during install, how can I do it now?
---------------------------------------------------------------------------------------------
-
-Run the following command (beware it can take a while, the download size is over 500 MB):
-
-``kalite manage unpack_assessment_zip https://learningequality.org/downloads/ka-lite/0.15/content/assessment.zip -f``
-
-As an alternative, you can download the zip file from https://learningequality.org/downloads/ka-lite/0.15/content/assessment.zip and run the command on your local copy. This is useful if you're deploying KA Lite on multiple computers.
-
 I can't get KA Lite to work on Windows! The installation succeeded, but nothing happens!
 ----------------------------------------------------------------------------------------
 
 KA Lite on Windows is controlled through a task-tray program.
 See the :doc:`installation guide <installguide/install_all>` for some more info.
+
+
+I can't see videos in Firefox on Ubuntu/Debian!
+-----------------------------------------------
+
+Install `Ubuntu restricted extras package <https://apps.ubuntu.com/cat/applications/ubuntu-restricted-extras/>`_ in Ubuntu Software Center.
```

Comment:
I wouldn't be me if I didn't eat an article or two... :stuck_out_tongue_winking_eye: